### PR TITLE
Assign multiple members to a group task

### PR DESCRIPTION
**Pull request by pihucode**
### Summary
This pull request allows multiple group members to be assigned to one group task and fixes some issues relating to the search box.

- [x] group task can be assigned to multiple group members
- [x] search input queries are no longer case-sensitive

### Test Plan
Played around with assigning group tasks to one or more group members through the task creator.

assign multiple members

### Notes
Task creator still cannot be closed when opened through the 'Assign new task' button.

### CODE REVIEW

File: frontend/src/components/GroupView/RightView/GroupTaskRow.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import ProfileImage from '../MiddleBar/ProfileImage';
 type Props = {
   readonly memberName: string;
   readonly userProfile: SamwiseUserProfile;
-  readonly onClick: (member: SamwiseUserProfile) => void;
+  readonly onClick: (member: SamwiseUserProfile[]) => void;
```
Comment (SamChou19815) [inline frontend/src/components/GroupView/RightView/GroupTaskRow.tsx:12]: make this array readonly

File: frontend/src/components/TaskCreator/GroupMemberPicker.tsx [hunk 2]
```diff
@@ -33,8 +33,29 @@ export default function GroupMemberPicker({
...
     onMemberChange(undefined);
-    if (clearAssignedMember) clearAssignedMember();
+    if (clearAssignedMembers) clearAssignedMembers();
   };
+  const getMemberNames = (selectedMembers: readonly SamwiseUserProfile[]): string => {
+    let memberNames = '';
+    let i = 0;
+    while (memberNames.length < 12 && i < selectedMembers.length) {
+      const firstName = selectedMembers[i].name.split(' ')[0];
+      if (memberNames.length + firstName.length < 12) {
```
Comment (SamChou19815) [inline frontend/src/components/TaskCreator/GroupMemberPicker.tsx:43]: why number 12?

File: frontend/src/components/TaskCreator/index.tsx [hunk 3]
```diff
@@ -130,10 +130,19 @@ export class TaskCreator extends React.PureComponent<Props, State> {
...
+    const { member, name, tag, date, complete, inFocus, subTasks } = this.state;
+
     if (name === '') {
       return;
     }
+
+    if (member) {
+      const newOwners = member.map((profile) => profile.email);
+      owner = newOwners;
+      console.log(owner);
```
Comment (SamChou19815) [inline frontend/src/components/TaskCreator/index.tsx:143]: no console.log

File: frontend/src/components/TaskCreator/GroupMemberPicker.tsx (hunk 2 above)
Comment (pihucode) [reply frontend/src/components/TaskCreator/GroupMemberPicker.tsx:43]: It's to limit the amount of characters displayed on the 'assign to' button.

**Review by SamChou19815 — APPROVED** [review]

**Review by SamChou19815 — APPROVED** [review]

**Pull request merged**